definitions/infra-prometheus-redis/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: INFRA
```

Comment:
Just to note: Registering under this domain implies that any 1st party redis entity types generated will have to abide by same domainID generation.

Reply:
@srvaroa have we decided that this should fall under EXT?

Reply:
Yes I think we agreed on that with the infra team. 

@kidk could you update the domain to EXT, and  change the directory name to "ext-redis"? (the directory is formed by concatenating domain-type.

Thanks!